### PR TITLE
Remove unused code pkg/controller/volume/expand

### PR DESCRIPTION
**Pull request by ZP-AlwaysWin**
**What type of PR is this?**

> /kind cleanup

**What this PR does / why we need it**:
This fixes golint failures of the following file:

- pkg/controller/volume/expand

**Which issue(s) this PR fixes**:

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```
None
```

### CODE REVIEW

**Comment by ZP-AlwaysWin** [comment]
/assign @draveness

I'm sorry, Git is not very familiar with it.
Re-modify it to meet your requirements
“use rebase instead of merge, and squash the commit into one.”

File: pkg/controller/volume/expand/pvc_populator.go [hunk 1]
```diff
@@ -56,7 +56,7 @@ type pvcPopulator struct {
 	recorder        record.EventRecorder
 }
 
-func NewPVCPopulator(
+func newPVCPopulator(
 	loopPeriod time.Duration,
```
Comment (gnufied) [inline pkg/controller/volume/expand/pvc_populator.go]: Actually - this file should be deleted. It was a mistake on my part to leave this file around. Can you do that please?

Comment (ZP-AlwaysWin) [reply pkg/controller/volume/expand/pvc_populator.go]: Yes，I will do it

**Comment by gnufied** [comment]
Did you run `./hack/update-bazel.sh` ?

**Comment by ZP-AlwaysWin** [comment]
> Did you run `./hack/update-bazel.sh` ?

Yes,I run ./hack/update-bazel.sh again

**Comment by gnufied** [comment]
@ZP-AlwaysWin can you squash the commits?

**Comment by ZP-AlwaysWin** [comment]
> @ZP-AlwaysWin can you squash the commits?

Have done it, could you review again. Thanks

**Comment by ZP-AlwaysWin** [comment]
> @ZP-AlwaysWin can you squash the commits?
hi @gnufied 
I have squash the commits, could you review again. Tannks

Best Wieshes

**Pull request merged**